### PR TITLE
[ Update ] Add description prop & slot to SfRadio

### PR DESCRIPTION
**Pull request by bahdcoder**
# Related issue

https://github.com/DivanteLtd/storefront-ui/issues/126

# Scope of work

```
    - add description option to SfRadio options prop
    - add [slot description] to SfRadio stories
    - add tests for description option
    - add description slot
```

# Screenshots of visual changes
Screen Shot 2019-06-14 at 1 18 24 PM

# Checklist

- [x] I followed [composition rules](https://github.com/DivanteLtd/storefront-ui/blob/master/docs/component-rules.md) for my component
- [ ] I tested the component in most common device sizes (can be tested in Storybook [from viewport addon in top menu](https://github.com/storybooks/storybook/tree/master/addons/viewport))

### CODE REVIEW

**Comment by filrak** [comment]
Hey! On the designs description is more gray-ish. Could you please use figma to check colors and find corresponding scss var?

You can find css values in css panel when clicking on a component in Figma 
image

File: packages/vue/src/components/molecules/SfRadio/SfRadio.html [hunk 1]
```diff
@@ -13,9 +13,18 @@
...
-        </slot>
+        <div class="sf-radio__content">
+            <!--@slot slot for customized option content layout -->
+            <slot name="content" :option="option">
+                <span class="sf-radio__content__label">{{ option.label }}</span>
+            </slot>
+
+            <!--@slot slot for customized option description layout -->
+            <slot name="description" :option="option" :isActive="option.value === value">
+                <p v-if="option.description" class="sf-radio__content__description">
```
Comment (filrak) [inline packages/vue/src/components/molecules/SfRadio/SfRadio.html]: It's not correct BEM naming

**Comment by bahdcoder** [comment]
@filrak PR updated

**Comment by filrak** [comment]
@bahdcoder styling is still different than on designs
image
vs
image

**Comment by bahdcoder** [comment]
Hi @filrak what differences do you think can be made ? At the moment all I see is the font.

**Comment by filrak** [comment]
@bahdcoder sorry for being not specific enough.

I meant font-size and color of the font

**Comment by filrak** [comment]
@bahdcoder I'll finish it by myself if you don't mid ;)

**Comment by bahdcoder** [comment]
@filrak I believe all properties match the design now. Please have another look

**Pull request merged**